src/RpcClient/Models/RpcContractState.cs

```diff
@@ -22,7 +23,7 @@ public static RpcContractState FromJson(JObject json)
                 Id = (int)json["id"].AsNumber(),
                 UpdateCounter = (ushort)json["updatecounter"].AsNumber(),
                 Hash = UInt160.Parse(json["hash"].AsString()),
-                Script = Convert.FromBase64String(json["script"].AsString()),
+                Nef = Convert.FromBase64String(json["script"].AsString()).AsSerializable<NefFile>(),
```

Comment:
https://github.com/neo-project/neo/blob/83857d232e755cbc3848a9158e96b54cdfcedb09/src/neo/SmartContract/ContractState.cs#L42-L52

There is no script field any more.

```
        public JObject ToJson()
        {
            return new JObject
            {
                ["id"] = Id,
                ["updatecounter"] = UpdateCounter,
                ["hash"] = Hash.ToString(),
                ["nef"] = Nef.ToJson(),
                ["manifest"] = Manifest.ToJson()
            };
        }
```